### PR TITLE
new template: Windows Azure VM as docker host with traefik and portainer pre-installed

### PR DESCRIPTION
**Pull request by tfenster**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog
* initial version

### CODE REVIEW

**Comment by tfenster** [comment]
@bmoore-msft Sorry if this is documented somewhere but I couldn't find it: In my template I run a script through a VM extension which downloads files from the repo. The URLs are defined [here ](https://github.com/tfenster/azure-quickstart-templates/blob/062fb6840f493c5b96815c86c2b10476db5482c5/docker-portainer-traefik-windows-vm/azuredeploy.json#L90-L91) and the download happens [here ](https://github.com/tfenster/azure-quickstart-templates/blob/062fb6840f493c5b96815c86c2b10476db5482c5/docker-portainer-traefik-windows-vm/setup.ps1#L24) and [here](https://github.com/tfenster/azure-quickstart-templates/blob/062fb6840f493c5b96815c86c2b10476db5482c5/docker-portainer-traefik-windows-vm/setup.ps1#L44). Because the PR is not yet merged, the files can't be downloaded and therefore the deployment fails. How can I get around this problem?

File: docker-portainer-traefik-windows-vm/Readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Windows Docker Host with Portainer and Traefik pre-installed
```
Comment (bmoore-msft) [inline docker-portainer-traefik-windows-vm/Readme.md:1]: change the casing on this file to README.md

File: docker-portainer-traefik-windows-vm/createUiDefinition.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
+{
```
Comment (bmoore-msft) [inline docker-portainer-traefik-windows-vm/createUiDefinition.json:1]: see pipeline results for changes that are needed here - also, you need to output the control values so they are used in deployment.

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: docker-portainer-traefik-windows-vm/createUiDefinition.json (hunk 2 above)
Comment (tfenster) [reply docker-portainer-traefik-windows-vm/createUiDefinition.json:1]: sorry, I didn't see the failures

**Comment by tfenster** [comment]
@bmoore-msft I hope I got it all sorted now, could you please take a look again?

**Pull request merged**